examples/Example9/example9.cu

```diff
@@ -252,13 +264,24 @@ void example9(const vecgeom::cxx::VPlacedVolume *world, int numParticles, double
   int iterNo = 0, loopingNo = 0;
   int previousElectrons = -1, previousPositrons = -1;
 
+  NVTXTracer tracer("SimulationLoop");
+  NVTXTracer occupTracer("LowOccup");
   do {
     Secondaries secondaries = {
         .electrons = {electrons.tracks, electrons.slotManager, electrons.queues.nextActive},
         .positrons = {positrons.tracks, positrons.slotManager, positrons.queues.nextActive},
         .gammas    = {gammas.tracks, gammas.slotManager, gammas.queues.nextActive},
     };
 
+    const auto nInFlight = std::accumulate(std::begin(stats->inFlight), std::end(stats->inFlight), 0);
+    if (nInFlight > 3400) {
+      occupTracer.setTag("PeakOccup");
+    } else if (nInFlight > 1000) {
+      occupTracer.setTag("HighOccup");
+    } else {
+      occupTracer.setTag("LowOccup");
+    }
```

Comment:
Just use `inFlight` :smiley:

Reply:
I didn't see it at this point. It's indeed there, but uninitialised on the first run. I'll set it to `numParticles`, and then it should be there for the following runs.